src/main/java/edu/wpi/grip/core/events/SocketConnectedChangedEvent.java

```diff
@@ -0,0 +1,36 @@
+package edu.wpi.grip.core.events;
+
+import com.google.common.base.MoreObjects;
+import edu.wpi.grip.core.Socket;
+
+/**
+ * This event is sent when the state of a socket (connected vs. disconnected) is changed.  Specifically, this will only
+ * be called if the number of connections either goes to zero when it was non-zero, or when it was zero and one is
+ * added.
+ * <p>
```

Comment:
I don't think so.  IntelliJ automatically adds a `<p>`, and it looks like that has the same effect.